### PR TITLE
Pr locale ru basic concepts translation

### PR DESCRIPTION
**Pull request by gitpash**
**What**:
I'm just add ru_locale for /basics section of glamorous website

**Why**: 
Glamorous is great tool, so I'd like to more people use it

**How**: 
I was working hard on each letter :)

So as we are two(with @IgorKonovalov ), working on ru_locale now, we'll collaborate and making code review, to prevent any issues. Already setup Trello for tasking. Thanks a lot Kent, for the great job, I'll do my best to be helpful.

### CODE REVIEW

**Comment by kentcdodds** [comment]
Thank you @gitpash! @IgorKonovalov could you review this?

File: pages/basics/content/ru/core-concepts.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+// применённые стили:
+// {margin: 1, fontSize: 2, padding: 1}
+// так же как любые применённые кастомно-классовые стили 
+```
+
+##### использование `css`
+
+Это может быть `css` того же типа, что и любой из переданных стилей
+(как в `glamorous.div(...styles)`). Если он указан, то будет объединён со стилем
+этого компонента, и получит высший приоритет среди всех предопределённых стилей
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/core-concepts.md]: компонента и

File: pages/basics/content/ru/core-concepts.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+с использованием `glamorous()` функции.
+
+```js
+const MyComposedStyledDiv = glamorous(MyStyledDiv)({fontSize: 4, padding: 4})
+<MyComposedStyledDiv />
+// применённые стили:
+// {margin: 1, fontSize: 4, padding: 4}
+```
+
+Фактически, встроенные фабрики DOM компонент, всего лишь абстракции этой функции,
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/core-concepts.md]: компонентов - всего лишь ... glamorous.div может быть ...

File: pages/basics/content/ru/dynamic-styles.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
+---
+title: Динамические Стили + Статические Стили
+---
+
+Одно из достоинств glamorous - это то, что он позволяет вам проводить чёткое разделение
+между динамическими и статическими стилями, принуждая вас выбирать между литеральным объектом
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/dynamic-styles.md]: просто объектом

File: pages/basics/content/ru/install.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+title: Установка
+---
+
+Этот модуль распространяется через [npm](https://www.npmjs.com/), связанного с [node](https://nodejs.org) и должен быть
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/install.md]: связанного -> предоставляемый

File: pages/basics/content/ru/install.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+---
+title: Установка
+---
+
+Этот модуль распространяется через [npm](https://www.npmjs.com/), связанного с [node](https://nodejs.org) и должен быть 
+установлен, как одна из зависимостей (`dependencies`) проекта:
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/install.md]: установлен как 
('dependencies') не надо - тавтология

File: pages/basics/content/ru/install.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+const {ThemeProvider} = glamorous
+// итд.
+```
+
+Если вы используете transpiling(и/или используете jsnext:main):
+
+```js
+import glamorous, {ThemeProvider} from 'glamorous'
+
+// вы можете импортировать нужный Glamorous Components (подробнее в разделе "Встроенные компоненты")
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/install.md]: Glamorous Компонент

File: pages/basics/content/ru/install.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+import {Div, H2} from 'glamorous'
+
+// тэги с тем же именем, что и встроенные объекты JavaScript, импортируются с Tag суффиксом
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/install.md]: суффиксом.
Имя тега, ..

File: pages/basics/content/ru/motivation.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
+---
+title: Мотивация
+subtitle: Зачем нужен glamorous?
+contributors:
+  - kentcdodds
+---
+
+### Проблема
+
+Вы любите CSS в JS, но вам не нравится необходимость создавать функции компонента, лишь для целей стилизации. Вы не хотите
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/motivation.md]: компонента лишь для стилизации

File: pages/basics/content/ru/motivation.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+придумывать имя для чего-то, относящегося только к стилям. Всё это очень раздражает: создание отдельных стилей, присваивание `className` и связанные с props'ами "танцы с бубном".
+
+Например, как это можно сделать только с помощью `glamor` (или `aphrodite` или чего-то подобного):
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/motivation.md]: Пример, как
сделать с помощью

File: pages/basics/content/ru/motivation.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+      className={`${styles} ${className}`}
+      {...rest}
+    />
+  )
+}
+```
+
+### Решение
+
+С `glamorous`, то, что выше, будет выглядеть просто как:
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/motivation.md]: первая запятая не нужна

File: pages/basics/content/ru/motivation.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+```js
+const MyStyledDiv = glamorous.div({
+  fontSize: 20,
+  textAlign: 'center',
+})
+```
+
+Фактически, даже лучше, из-за множества возможностей, которые делают составление компонент друг с другом приятным!
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/motivation.md]: тут нужно перефразировать, ничего не понятно

File: pages/basics/content/ru/motivation.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+    >
+      Hello world!
+    </Div>
+  )
+}
+```
+
+> Попробовать в браузере [здесь](https://codesandbox.io/s/mDLZ1oKn)!
+
+Итак, это основы такого решения... Продолжим знакомиться с деталями!
```
Comment (IgorKonovalov) [inline pages/basics/content/ru/motivation.md]: основы рещения 'glamorous'...

**Comment by paulmolluzzo** [comment]
@gitpash This is great, thank you!

I'm going to merge this so it gets out there ASAP, but you might want to open a PR to add your name as a `contributor` in the front matter of the markdown files. [Look at this file for an example](https://raw.githubusercontent.com/kentcdodds/glamorous-website/4430f3e961fba7c6d5a4569677a2c926ebf9735a/pages/basics/content/motivation.md).

**Comment by kentcdodds** [comment]
Whoops I just merged @IgorKonovalov's PR Anand I guess there are merge conflicts. Could you redone those. Sorry. Let us know if you need help.

**Comment by paulmolluzzo** [comment]
Whoops, there are conflicts in the README and `.all-contributorsrc`. Can you please resolve those @gitpash?

**Comment by paulmolluzzo** [comment]
@kentcdodds is a step ahead of me. 😆

**Comment by kentcdodds** [comment]
For the readme, don't bother. Just fix the .all-contributorsrc and run `npm start contrib.generate` and that'll remembered the README 👍

**Comment by gitpash** [comment]
Ok, try again😆

**Comment by kentcdodds** [comment]
Thanks!

**Pull request merged**